### PR TITLE
Add getting involved section to the readme

### PR DESCRIPTION
**Pull request by robtaylor**
Noticed that the comms channels for OpenRoad weren't immediately obvious to a newcomer, so added this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,17 @@ includes GlobalFoundries shuttles, design contests and IC design
 workshops. The OpenROAD flow has been successfully used to date
 in over 600 silicon-ready tapeouts for technologies up to 12nm.
 
+## Getting Involved
+
+There is a users mailing list, which you can join [here](https://groups.google.com/g/openroad-users).
+
+There is also a [Matrix](https://matrix.org/) channel for discussion at 
+[#openroad:matrix.org](https://matrix.to/#/%23openroad:matrix.org).
```
Comment (maliberty) [inline README.md:116]: I'm also unfamiliar with this service.  It appears to require an account to even see the content so I don't know how relevant it is.  I don't believe anyone from the OR team interacts here.  Do you see a lot of useful content here?

File: README.md [hunk 2]
```diff
@@ -108,6 +108,17 @@ includes GlobalFoundries shuttles, design contests and IC design
 workshops. The OpenROAD flow has been successfully used to date
 in over 600 silicon-ready tapeouts for technologies up to 12nm.
 
+## Getting Involved
+
+There is a users mailing list, which you can join [here](https://groups.google.com/g/openroad-users).
```
[the remark this answers is not in the document]
Comment (oharboe) [reply README.md:113]: Never heard of it :-)

File: README.md (hunk 1 above)
Comment (oharboe) [reply README.md:116]: Being a social media ludite, haven't heard of it...

**Comment by robtaylor** [comment]
Matrix is common open messaging service used by many open source projects.
It has been gradually replacing or being bridged with irc for a few years
now.

It’s an open standard, with a number of open source clients.

OpenRoad users was the only mailing list I could find to add here. Is there
another? If not I suggest the project grows one.

**Comment by maliberty** [comment]
Most conversations happen in GH or slack.  If there is organic demand for a mailing list I'm open but I haven't see any so far.

**Comment by maliberty** [comment]
I don't see either of these as forums to promote at this point.

**Comment by robtaylor** [comment]
I think any forum would be useful to promote.

What do you consider the correct forum to be?

**Comment by oharboe** [comment]
I dont know what "correct forum" means, but currently there is a little bit of activity in github discussions under ORFS and OpenROAD... It has good participation from maintainers, hard to compete with... Strangely there is no such thing as github chat&chat rooms....

**Comment by robtaylor** [comment]
Yeah, that makes sense. I’ll fix the patch to cover that instead.

I really think a matrix or irc room would be very helpful for contributors
and users - these can be the same room and for most OSS projects that’s the
common practice.

Obviously there’s the open-silicon-dev.org slack room as well, but I think
we’d all agree a proprietary chat platform is inappropriate.

(Again just for your info, matrix is a an open federated protocol, which
gives slack-like capabilities. There are bridges to many other chat-like
protocols.)

**Comment by robtaylor** [comment]
I’ll just add that we could bridge to the slack room if that room is seen
as useful/important. (
https://matrix.org/ecosystem/bridges/slack/)

**Comment by maliberty** [comment]
Most chit-chat is on open-source-silicon.slack.com

**Pull request closed without merging**